### PR TITLE
Hotfix/filename

### PR DESCRIPTION
**Pull request by Shahar-Y**
Changed the way filename is referenced.

### CODE REVIEW

File: src/file/file.model.ts [hunk 1]
```diff
@@ -13,13 +14,10 @@ export const fileSchema: mongoose.Schema = new mongoose.Schema(
       sparse: true,
       unique: true,
     },
-    displayName: {
+    name: {
```
Comment (omrishtam) [inline src/file/file.model.ts]: Should define unique index on `name` and `parent` fields.

File: src/file/file.model.ts [hunk 2]
```diff
@@ -3,6 +3,7 @@ import { ServerError } from '../utils/errors/application.error';
 import { IFile } from './file.interface';
 import { KeyAlreadyExistsError } from '../utils/errors/client.error';
 import { MongoError } from 'mongodb';
+import { NextFunction } from 'connect';
 
 const ObjectId = mongoose.Schema.Types.ObjectId;
```
Comment (omrishtam) [inline src/file/file.model.ts]: Don't import the whole mongoose package just for ObjectId and other parts of the package.

File: src/file/file.model.ts (hunk 1 above)
Comment (Shahar-Y) [reply src/file/file.model.ts]: why not also owner?
there could be two files with the same name whose parent is null (root), but the owner is different

Comment (Shahar-Y) [reply src/file/file.model.ts]: and that makes another problem - what if the file is not owned by me, but is on the root and has the same name as the other file in root?

Comment (omerzamir) [reply src/file/file.model.ts]: You are absolutely right.

Comment (omrishtam) [reply src/file/file.model.ts]: You are right, add that too to the unique index.

Comment (omrishtam) [reply src/file/file.model.ts]: This isn’t possible, if a file is in a root folder it has to be under a certain bucket. Either it’s yours or not.

**Comment by Shahar-Y** [comment]
@omrishtam @omerzamir @AllKinds 
All checks pass and the branch is merged with develop. 
Please re-check the PR so I can continue working on the other issues.

**Comment by omrishtam** [comment]
Looks good to me, but don’t merge it there are api changes like field names etc.
If there aren’t such changes then I think you could merge the PR, otherwise comment what changes have been made to the api so that we can update our dependent services (api-gw)

**Comment by omrishtam** [comment]
Didn’t mean you would merge immediately🙃

**Pull request merged**